Added JSON support for reading rules and selectors from pipeline (#858)
microsoft/PSRule: docs/concepts/PSRule/en-US/about_PSRule_Selectors.md

### Defining selectors

Selectors can be defined with either YAML or JSON format, and can be included with a module or standalone `.Rule.yaml` or `.Rule.jsonc` file.
In either case, define a selector within a file ending with the `.Rule.yaml` or `.Rule.jsonc` extension.
A selector can be defined side-by-side with other resources such as baselines or module configurations.

> Use `.jsonc` to view [JSON with Comments](https://code.visualstudio.com/docs/languages/json#_json-with-comments) in Visual Studio Code.

Reviewer: ```suggestion
Selectors can also be defined within `.json` files.
We recommend using `.jsonc` to view [JSON with Comments](https://code.visualstudio.com/docs/languages/json#_json-with-comments) in Visual Studio Code.
```

    - 'Value2'
```

> ### Example Selectors.Rule.json

Reviewer: ```suggestion
### Example Selectors.Rule.jsonc
```

```json

> // Example Selectors.Rule.json

Reviewer: ```suggestion
// Example Selectors.Rule.jsonc
```
